src/payment.rs

```diff
@@ -26,8 +26,38 @@ pub fn credit_card_number() -> String {
 }
 
 pub fn credit_card_luhn_number() -> String {
-    // @TODO
-    return String::from("");
+    let mut cc_str = credit_card_number();
```

Comment:
What do you think writing the algorithm of luhn generation? I think it's much nicer solution to write the function on the way that it generates a proper luhn number sequence for the first run.

Resources:
- https://www.groundlabs.com/blog/anatomy-of-a-credit-card/
- https://atufashireen.medium.com/luhn-algorithm-67c62e081238

Let me know your thoughts. :)

Reply:
Yeah, I absolutely think that is the right way to go.  I only did it this way because I was trying to stay close to the go fakeit library, but if you're cool diverging a bit, then generating the last digit is much better and potentially future-usable than guess-and-check.

Will change.

Reply:
Yes, I'm not a big fan of how gofakeit solves it, so that would be awesome if you could have change it. Thank you.